### PR TITLE
refactor: migrate organization screen to styled-components

### PR DESCRIPTION
**Pull request by ZahraTee**
## Screenshots

| Before   | After    |
| -------- | -------- |
|before|after|

Part of #532.

### CODE REVIEW

**Comment by machour** [comment]
Since this is affecting the UI, could you provide us with two screenshots of the organization screen?
(One without the PR, and one with this PR applied)

File: src/organization/screens/organization-profile.screen.js [hunk 1]
```diff
@@ -47,16 +48,16 @@ const actionCreators = {
 
 const actions = dispatch => bindActionCreators(actionCreators, dispatch);
 
-const styles = StyleSheet.create({
-  listTitle: {
+const DescriptionListItem = styled(ListItem).attrs({
+  titleStyle: {
```
Comment (machour) [inline src/organization/screens/organization-profile.screen.js]: I think this one can be removed (unused)

**Comment by ZahraTee** [comment]
Added the screenshots :)

**Comment by machour** [comment]
LGTM, thanks!

File: src/organization/screens/organization-profile.screen.js [hunk 2]
```diff
@@ -47,16 +48,12 @@ const actionCreators = {
...
-    ...fonts.fontPrimary,
-  },
-  listSubTitle: {
+const DescriptionListItem = styled(ListItem).attrs({
+  subtitleStyle: {
     color: colors.greyDark,
     ...fonts.fontPrimary,
   },
-});
+})``;
```
Comment (andrewda) [inline src/organization/screens/organization-profile.screen.js]: Can we remove this? Was probably just added by `prettier` for some reason, but I think you can safely get rid of it.

Comment (machour) [reply src/organization/screens/organization-profile.screen.js]: It's needed by styled components when using `attrs()`

Comment (andrewda) [reply src/organization/screens/organization-profile.screen.js]: Ahh alright, yes you're right.

**Pull request merged**